tests/invert_test.cpp

```diff
@@ -47,6 +48,7 @@ void display_test_info()
     printfQuda(" - number of levels %d\n", mg_levels);
     for (int i = 0; i < mg_levels - 1; i++) {
       printfQuda(" - level %d number of null-space vectors %d\n", i + 1, nvec[i]);
+      printfQuda(" - level %d null-space vector batch size %d\n", i + 1, nvec_batch[i]);
```

Comment:
This needs to be propagated into `staggered_invert_test.cpp`

Reply:
f0d9f3c8dc79bdb4ed751b34193262c517a6b287